package/gluon-web-private-wifi/i18n/de.po

```diff
@@ -1,38 +1,45 @@
-msgid ""
+msgid "8-63 characters"
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: 2015-05-04 02:25+0200\n"
-"Last-Translator:  <mschiffer@universe-factory.net>\n"
-"Language-Team: German\n"
-"Language: de\n"
-"MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-msgid "8-63 characters"
-msgstr "8-63 Zeichen"
+msgid "Disabled"
+msgstr ""
 
 msgid "Enabled"
-msgstr "Aktiviert"
+msgstr ""
+
+msgid "Encryption"
+msgstr ""
 
 msgid "Key"
-msgstr "Schlüssel"
+msgstr ""
+
+msgid "Management Frame Protection"
+msgstr ""
 
 msgid "Name (SSID)"
-msgstr "Name (SSID)"
+msgstr ""
+
+msgid "Optional"
+msgstr ""
 
 msgid "Private WLAN"
-msgstr "Privates WLAN"
+msgstr ""
+
+msgid "Required"
+msgstr ""
+
+msgid "WPA2"
+msgstr ""
+
+msgid "WPA2 / WPA3"
+msgstr ""
+
+msgid "WPA3"
+msgstr ""
 
 msgid ""
 "Your node can additionally extend your private network by bridging the WAN "
 "interface with a separate WLAN. This feature is completely independent of "
 "the mesh functionality. Please note that the private WLAN and meshing on the "
 "WAN interface should not be enabled at the same time."
 msgstr ""
-"Dein Knoten kann zusätzlich die Reichweite deines privaten Netzes erweitern. "
-"Hierfür wird der WAN-Port mit einem separaten WLAN gebridged. Diese "
-"Funktionalität ist völlig unabhängig von den Mesh-Funktionen des Knotens. "
-"Beachte, dass du nicht gleichzeitig das Meshen über den WAN-Port aktiviert "
-"haben solltest."
```

Comment:
why is this section being removed?

Reply:
Because I'm to stupid to use tools correctly. Fixed now.